view/theme/frio/templates/generic_links_widget.tpl

```diff
@@ -0,0 +1,12 @@
+
+<div class="widget{{if $class}} {{$class}}{{/if}}">
+	{{if $title}}<h3>{{$title}}</h3>{{/if}}
+	{{if $desc}}<div class="desc">{{$desc}}</div>{{/if}}
+
+	<ul role="menu">
+		{{foreach $items as $item}}
+			<li role="menuitem" class="{{if $item.selected}}selected{{/if}}"><a href="{{$item.url}}" {{if $item.accesskey}}accesskey="{{$item.accesskey}}"{{/if}}>{{$item.label}}</a></li>
```

Comment:
Can you please respect the HTML indentation when adding a template loop? What I mean is that in the resulting HTML, I'm expecting the `li` to be one indentation level further than the `ul`, not two.